Test.Microsoft.NET.Build.Containers.Filesystem/TestInitialization.cs

```diff
@@ -0,0 +1,17 @@
+namespace Test.Microsoft.NET.Build.Containers.Filesystem;
+
+[TestClass]
+public static class TestInitialization {
+
+    [AssemblyInitialize]
```

Comment:
You can only have one AssemblyInitialize in MSTest, so I've centralized all the setup/teardown logic here.